query_optimizer/rules/UnnestSubqueries.cpp

```diff
@@ -300,6 +302,7 @@ E::ExpressionPtr UnnestSubqueriesForNonRootLogical::eliminateOuterAttributeRefer
     std::vector<E::AttributeReferencePtr> *probe_join_attributes,
     std::vector<E::AttributeReferencePtr> *build_join_attributes,
     std::vector<E::PredicatePtr> *non_hash_join_predicates) {
+  DCHECK(expression->getExpressionType() != E::ExpressionType::kInTableQuery);
```

Comment:
It appears that DCHECK_NE cannot apply to `enum class` values because it uses the `<<` operator to print out the checked value -- however there is no `<<` overloading for the enum class.